server/ot_rpc.go

```diff
@@ -233,6 +235,13 @@ func (h *lphttp) TranscodeResults(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	decodedPixels, err := strconv.ParseInt(r.Header.Get("Pixels"), 10, 64)
```

Comment:
I was on the fence about just calling this header name "Pixels" vs. "EncodedPixels". I stuck with "Pixels" for now, but am open to changing it if people think the latter is clearer.